app/controllers/pseuds_controller.rb

```diff
@@ -14,7 +14,7 @@ def load_user
   # GET /pseuds.xml
   def index
     if @user
-      @pseuds = @user.pseuds
+      @pseuds = @user.pseuds.paginate(page: params[:page])
```

Comment:
Should we set this to use the `alphabetical` scope? I noticed when I was playing with tests I'd sometimes get the wrong pseud on the second page if I didn't add that.

Speaking of tests, I added this right after https://github.com/otwcode/otwarchive/blob/ba94c04580de2bac4c7a932e33a11fad4de7e06d/features/other_a/pseuds.feature#L162
```
  When I go to my pseuds page
  Then I should not see "Zaphod (Zaphod)" within "ul.pseud.index"
    But I should see "Agrajag (Zaphod)" within "ul.pseud.index"
    And I should see "Betelgeuse (Zaphod)" within "ul.pseud.index"
    And I should see "Slartibartfast (Zaphod)" within "ul.pseud.index"
    And I should see "Next" within ".pagination"
  When I follow "Next" within ".pagination"
  Then I should see "Zaphod (Zaphod)" within "ul.pseud.index"
```

Reply:
Sure! Do we want `alphabetical` or `default_alphabetical`? Presumably what we use now will stick for the foreseeable 🙈

Reply:
Thanks for the spec! 🙌

Reply:
alphabetical is good! That's what it seems to be on staging